templates/default/rt/rtticketinfobox.html

```diff
@@ -54,6 +54,7 @@
 									{$ticket.subject|escape|truncate:50:"...":true}
 								</TD>
 							</TR>
+							{if !$ticket.service == SERVICE_OTHER}
```

Comment:
Co to za dziwaczne warunki? Mało to czytelne - nawet jeśli jakoś fuksem działa.

Reply:
Dlaczego dziwaczne?

Idea jest taka - "Pokazuj jeśli inne niż domyślne".

Reply:
Chodzi o $ticket.service != SERVICE_OTHER? Jeśli tak to tak proponuję zapisać.